src/main/features/claimant-response/views/counter-offer-accepted.njk

```diff
@@ -0,0 +1,36 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, saveAndContinueButton, errorSummary, textArea %}
```

Comment:
remove textArea and errorSummary as you don't use them

Reply:
done